_docs/_latest/configure/scanner/descriptions.md

```diff
@@ -96,7 +96,7 @@ For examples of how to define scanner rules for your forwarding rules, see the
 [`forwarding_rules.yaml`](https://github.com/GoogleCloudPlatform/forseti-security/blob/stable/rules/forwarding_rules.yaml)
 rule file.
 
-## Groups scanner
+## G Suite scanner
```

Comment:
I think in the yaml file we are using `group` for the scanner and `groups_violations` for the violation, do you think it might confuse user if we change the doc to g suite scanner?

Reply:
Fair point, we are looking at more than groups though. We inventory: groups, group members, and domain users.

In the future we could have an auditor that looks at user violations and it could be user_violations? It seems renaming groups_violations to gsuite_violations isn't wise or trivial?

Reply:
Yes, I agreed it makes more sense to rename it to G Suite. Maybe it's better to do the renaming in the doc after we updated the Forseti code?